src/git-repo.ts

```diff
@@ -28,6 +28,16 @@ export class GitRepo {
     return true
   }
 
+  isSetup(): boolean {
```

Comment:
hi @ivanramosnet 

- Could you add a test for this new method [here](https://github.com/Nautilus-Cyberneering/git-queue/blob/main/__tests__/unit/git-repo.test.ts)?
- I would rename the method to `this.isAuthorIdentityUnknown` because there could be other setup stuff.
- And also for the error. I think we could use the same name Git uses: `AuthorIdentityUnknownErrror`